### PR TITLE
fix: add 'SET NULL' constraint to migration

### PR DESCRIPTION
**Pull request by woshilapin**
Should address comment https://github.com/CanalTP/kirin/pull/428#discussion_r761918635. I've applied the `NOT NULL` constraint on `dev` DB manually too (so this migration should not be played again on next deployment).

### CODE REVIEW

**Review by pbougue — CHANGES_REQUESTED** [review]
This should only be part of #426, it's fully related and only related to #426 PR (and #426 PR won't work without it).

**Review by pbougue — APPROVED** [review]
Ahhh, the mixup was partly in my head on this, it's better with this indeed.

File: migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py [hunk 1]
```diff
@@ -49,11 +49,13 @@ def downgrade():
 
     op.execute("ALTER TABLE real_time_update ADD COLUMN old_connector old_connector_type")
     op.execute("UPDATE real_time_update SET old_connector = connector::text::old_connector_type")
+    op.execute("ALTER TABLE realtime_update ALTER COLUMN old_connector SET NOT NULL")
```
Comment (pbougue) [inline migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py:52]:
```suggestion
    op.execute("ALTER TABLE real_time_update ALTER COLUMN old_connector SET NOT NULL")
```

File: migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py [hunk 2]
```diff
@@ -49,11 +49,13 @@ def downgrade():
...
     op.execute("ALTER TABLE real_time_update DROP connector")
     op.execute("ALTER TABLE real_time_update RENAME COLUMN old_connector TO connector")
 
     op.execute("ALTER TABLE contributor ADD COLUMN old_connector_type old_connector_type")
     op.execute("UPDATE contributor SET old_connector_type = connector_type::text::old_connector_type")
+    op.execute("ALTER TABLE realtime_update ALTER COLUMN old_connector_type SET NOT NULL")
```
Comment (pbougue) [inline migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py:58]:
```suggestion
    op.execute("ALTER TABLE real_time_update ALTER COLUMN old_connector_type SET NOT NULL")
```

File: migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py (hunk 1 above)
Comment (woshilapin) [reply migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py]: :facepalm:

File: migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py (hunk 2 above)
Comment (woshilapin) [reply migrations/versions/1639faa28bc4_add_siri_et_lite_idfm.py]: :facepalm:

**Pull request merged**